go/registry/api/runtime.go

```diff
@@ -107,6 +92,50 @@ type Runtime struct {
 
 	// KeyManager is the key manager runtime ID for this runtime.
 	KeyManager signature.PublicKey `json:"key_manager"`
+
+	// ReplicaAllowedStragglers is the number of allowed stragglers.
+	ReplicaAllowedStragglers uint64 `json:"replica_allowed_stragglers"`
```

Comment:
So other things were moved into either compute/merge/txn sched, should this be as well?